pkg/proxy/userspace/proxysocket.go

```diff
@@ -188,19 +188,19 @@ func (udp *udpProxySocket) Addr() net.Addr {
 }
 
 // Holds all the known UDP clients that have not timed out.
-type clientCache struct {
-	mu      sync.Mutex
-	clients map[string]net.Conn // addr string -> connection
+type ClientCache struct {
+	sync.Mutex
```

Comment:
I think it's just convention - it's documented at https://github.com/kubernetes/community/blob/master/contributors/devel/coding-conventions.md, which also says it should be called "lock".  My C++ tingles in exposing a lock, which is why I suggested method-wrapping.